app/modules/win32ipc/module.cc

```diff
@@ -6,17 +6,24 @@
 
 #include "pipe_wrapper.h"
 
+#define NORMAL_STRING v8::String::NewStringType::kNormalString
+v8::Local<v8::String> newFromUtf8(v8::Isolate* isolate, char const* str) {
```

Comment:
newStringFromUtf8 is likely a better name (since it's no longer being called prepended by the class name)